code/game/objects/items/devices/modkit.dm

```diff
@@ -314,6 +314,34 @@
 		/obj/item/clothing/suit/space/void/unathi_pirate/captain = /obj/item/clothing/suit/space/void/unathi_pirate/hiskyn/captain
 	)
 
+/obj/item/voidsuit_modkit/heph_unathi
```

Comment:
We have a refitting system. Is this needed?

Reply:
There is not a cycler for these voidsuits and their status as uplink items means that I want refitting them to be accessible to antags.